### DISCUSSION TITLE
plugin update with vim-plug fails

### DISCUSSION THREAD

**Discussion by minusf** [nvim-tree/nvim-tree.lua#1903] [Q&A]
```
x nvim-tree.lua:
    fatal: Not possible to fast-forward, aborting.
```

**Comment by alex-courtis** [comment]
Please accept my apologies #1906

Please remove nvim-tree and reinstall.

**Comment by minusf** [comment]
git is difficult indeed.  today i am getting this:

```
x nvim-tree.lua:
    fatal: refusing to merge unrelated histories
```

**Reply by alex-courtis** [reply]
Try doing a full removal and reinstall: please delete the nvim-tree.lua folder from the file system then run a `:PlugInstall`
